### PR TITLE
Add tests for the operation(install/uninstall/update package) when global settings file is corrupted

### PR DESCRIPTION
**Pull request by GangWang01**
https://github.com/dotnet/templating/issues/5972 - Add tests for the change of improved error message in https://github.com/dotnet/templating/pull/6191

### CODE REVIEW

**Comment by GangWang01** [comment]
@vlada-shubina @YuliiaKovalova Do you think the error message in the approvals is acceptable, especially the ones for uninstalling/updating package?

**Comment by vlada-shubina** [comment]
> @vlada-shubina @YuliiaKovalova Do you think the error message in the approvals is acceptable, especially the ones for uninstalling/updating package?

it's unfortunate that message is shown multiple items, but i'm not sure we want to do anything for this at this point. What was the user experience before your change?

**Comment by GangWang01** [comment]
> > @vlada-shubina @YuliiaKovalova Do you think the error message in the approvals is acceptable, especially the ones for uninstalling/updating package?
> 
> it's unfortunate that message is shown multiple items, but i'm not sure we want to do anything for this at this point. What was the user experience before your change?

It's similar to this. Just the error message is different, but appears multiple times for updating/uninstalling package.

**Comment by YuliiaKovalova** [comment]
@vlada-shubina ,
is it worth investigating how to dedupe these warning messages or it is better to make a part of a separate issue?

File: src/Tests/dotnet-new.Tests/DotnetNewUpdateTests.Approval.cs [hunk 1]
```diff
@@ -28,5 +28,29 @@ public Task CanShowMessageInCaseShortNameConflict()
...
+                .WithoutBuiltInTemplates()
+                .Execute()
+                .Should()
+                .Pass()
+                .And.HaveStdOutContaining("console");
+
+            var globalSettingsFile = Path.Combine(homeDirectory, "packages.json");
+            File.WriteAllText(globalSettingsFile, string.Empty);
+
+            CommandResult commandResult = new DotnetNewCommand(_log, "uninstall", "TemplateWithRequiredParameters")
```
Comment (YuliiaKovalova) [inline src/Tests/dotnet-new.Tests/DotnetNewUpdateTests.Approval.cs:47]: is it expected to run "uninstall" in DotnetNewUpdateTests class?

**Comment by YuliiaKovalova** [comment]
> Let's keep it as separate issue, definitely low prio.

Can we deliver these changes having multiple warnings displayed?

**Review by YuliiaKovalova — APPROVED** [review]

**Pull request merged**